src/components/Settings/SettingsNetwork/index.tsx

```diff
@@ -104,6 +186,19 @@ const SettingsNetwork = () => {
           validationSchema={NetworkSettingsSchema}
           onSubmit={async (values) => {
             try {
+              const trustedProxies: { v4: string[]; v6: string[] } = {
+                v4: [],
+                v6: [],
+              };
+              for (let value of values.trustedProxies.split(',')) {
+                value = value.trim();
+                if (value.indexOf('.') != -1) {
```

Comment:
Since we have `ip-address`, we can use `isValid` instead of `indexOf` to perform proper validation.

Reply:
@M0NsTeRRR This is not doing validation. It is just for separating v4 and v6 addresses for pretty printing in the text field. We do not have to validate them.

If they are invalid, Right after they are rendered to the input, User will see an address validation error

Reply:
The ip-address library operates on strings/regex rather than constructing an int or [int;2] and have fast implementations for these operations. This will be hit on every request and I didn't want the needless validation to slow things down.

Reply:
Understable !